task3/task3/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("x=");
+            double x = Convert.ToDouble(Console.ReadLine());
```

Comment:
Старайтесь избегать использования класса Convert без особой необходимости. Лучше используйте методы имеющихся типов.
```suggestion
            double x = double.Parse(Console.ReadLine());
```